tap_playvox/sync.py

```diff
@@ -159,10 +186,19 @@ def sync_endpoint(client,
                         
                         if len(date_records)==parsed_date_records:
                             parse = False
-                            
+
+        # Subtract 14 days from the current date and time for Schedule Metrics
+        startDate = datetime.utcnow() - timedelta(days=14)
+
+        # Set the time to midnight
+        startDate = startDate.replace(hour=0, minute=0, second=0, microsecond=0)
```

Comment:
this affects all the streams. you should instead just do

```python
if stream == schedule_metrics:
    start_datetime -= timedelta(days=14)
...
data = client.get(...)
```

instead of modifying the bookmarks to be 14 days behind

Reply:
Good idea!

Reply:
Change made